### PR TITLE
Fix cacert issue

### PR DESCRIPTION
**Pull request by jichenjc**
cacert don't need to be base64 or it will be base64 encode twice and lead to incorrect format in cloud-init

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: examples/controlplane/controlplane.yaml [hunk 1]
```diff
@@ -258,9 +256,8 @@ spec:
   - path: /etc/certs/cacert
     owner: root
     permissions: "0600"
```
[the remark this answers is not in the document]
Comment (jichenjc) [reply examples/controlplane/controlplane.yaml]: 6 works for me :)

File: examples/generate.sh [hunk 2]
```diff
@@ -178,18 +178,20 @@ if [[ "$OS" =~ "Linux" ]]; then
...
     export OPENSTACK_CLOUD_CACERT_CONFIG
-    OPENSTACK_CLOUD_CACERT_CONFIG=$(cat "$CACERT_ORIGINAL"|base64 -w0)
+    OPENSTACK_CLOUD_CACERT_CONFIG=$(cat "$CACERT_ORIGINAL")
   fi
 elif [[ "$OS" =~ "Darwin" ]]; then
 #  export OPENSTACK_CLOUD_PROVIDER_CONF=$(echo "$OPENSTACK_CLOUD_PROVIDER_CONF_PLAIN"|base64)
   if [[ "$CACERT_ORIGINAL" != "null" ]]; then
     export OPENSTACK_CLOUD_CACERT_CONFIG
-    OPENSTACK_CLOUD_CACERT_CONFIG=$(cat "$CACERT_ORIGINAL"|base64)
+    OPENSTACK_CLOUD_CACERT_CONFIG=$(cat "$CACERT_ORIGINAL")
```
[the remark this answers is not in the document]
Comment (jichenjc) [reply examples/generate.sh]: yes, did some optimization

File: examples/machinedeployment/machinedeployment.yaml [hunk 3]
```diff
@@ -72,9 +72,8 @@ spec:
         owner: root
         path: /etc/kubernetes/cloud.conf
         permissions: "0600"
-      - content: |-
-          # cacert to communicate with OpenStack
-          $OPENSTACK_CLOUD_CACERT_CONFIG
+      - content: |
+          $OPENSTACK_CLOUD_CACERT_CONFIG6
```
Comment (sbueringer) [inline examples/machinedeployment/machinedeployment.yaml]: Did you try the machinedeployment? 6 spaces shouldn't be enough (I had to indent cloud provider config with 10)

Comment (jichenjc) [reply examples/machinedeployment/machinedeployment.yaml]: you are right, I forget the check this file after modification
10 used to work before my modification :(

**Pull request merged**